.github/workflows/commit-metadata-all-environments.yml

```diff
@@ -7,8 +7,9 @@
 name: Commit metadata changes from org for all environments
 
 on:
-  schedule:
-    - cron: "0 1 * * *" # Runs nightly at 01:00 UTC.
+  # Uncomment to run this workflow on a schedule:
+  #schedule:
+  #  - cron: "0 1 * * *" # Run nightly at 01:00 UTC.
```

Comment:
Might be wise to change this schedule because it clashes with the merge from production schedule

Reply:
What do you reckon we should change it to? 4 times throughout the working day? 9, 12, 15 and 18?

Reply:
Seems sensible